scm/src/main/java/com/sandboni/core/scm/resolvers/RevisionResolver.java

```diff
@@ -126,8 +126,5 @@ private void validate(ObjectId from, ObjectId to) throws SourceControlException
         if (from == null || to == null || from.equals(ObjectId.zeroId())) {
             throw new SourceControlException(ErrorMessages.UNABLE_TO_RESOLVE_REVISIONS);
         }
-        if (from.equals(to)) {
```

Comment:
Why is this validation not longer needed?

Reply:
We are allowing same to and from commits now by using the CLI change scope resolver